cats/shared/src/main/scala/caseapp/cats/CatsArgParser.scala

```diff
@@ -0,0 +1,14 @@
+package caseapp.cats
+
+import caseapp.core.argparser.{ArgParser, SimpleArgParser}
+import cats.data.NonEmptyList
+
+object CatsArgParser {
+  implicit def nonEmptyListArgParser[T]: ArgParser[NonEmptyList[T]] =
```

Comment:
Missing implicit `ArgParser[T]`?
```suggestion
  implicit def nonEmptyListArgParser[T: ArgParser]: ArgParser[NonEmptyList[T]] =
```